src/mpi/comm/builtin_comms.c

```diff
@@ -5,6 +5,9 @@
 
 #include "mpiimpl.h"
 
+#define COMM_WORLD_CTXID 0
```

Comment:
Should this be `0 << MPIR_CONTEXT_PREFIX_SHIFT`?

Reply:
They are the same and I somehow find `0` much clearer. But I don't mind change it.

Reply:
My instinct is leave it like before if you don't mind.

Reply:
Fixed.